_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,488 @@
+---
+id: be4d0c
+name: Inline link has either distinguishable style not based on color (hue) alone or content that indicates it is a link
+rule_type: atomic
+description: |
+  This rule checks that inline links are distinguishable from the surrounding text through a difference in style not based on color (hue) alone or have content that indicates they are links.
+accessibility_requirements:
+  wcag20:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [visible][] HTML element that is a [semantic link][], for which all the following is true:
+
+- **link text**: the element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
+- **non-link line text**: the element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][]; and
+- **different hue**: the element's [foreground colors][foreground color] and the [foreground color][] of the **non-link line text** elements have a [different hue][] in at least one [link history state][], or the element's [background colors][background color] and the [background color][] of the **non-link line text** elements have a [different hue][] in at least one [link history state][].
```

Comment:
> The Use of Color success criterion addresses changing only the color (hue) of an object or text without otherwise altering the object's form.

This is quoted from the [understanding document of SC 1.4.11](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)
After Jean-Yves noticed this, this rule was changed accordingly to focus on hue.
You can check this in the last assumption of the rule.